src/backend/function/Fusion.Resources.Functions/Deployment/function.template.json

```diff
@@ -27,7 +27,9 @@
         },
         "queues": {
           "provisionPosition": "provision-position",
-          "scheduledNotificationReportQueue": "scheduled-notification"
+          "scheduledNotificationReportQueue": "scheduled-notification",
+          "departmentSummaryWeeklyQueue": "department-summary-weekly-queue"
```

Comment:
Shouldnt this be added to the summary function template?
https://github.com/equinor/fusion-app-resources/blob/5ddaa3cdc0c66b244e8f18e8d2d35d9c398bc338/src/Fusion.Summary.Functions/Deployment/function.template.json